Core/Sources/CodeCompletionService/API/AnthropicService.swift

```diff
@@ -0,0 +1,235 @@
+import CopilotForXcodeKit
+import Foundation
+import Fundamental
+
+public actor AnthropicService {
+    let url: URL
+    let modelName: String
+    let contextWindow: Int
+    let maxToken: Int
+    let temperature: Double
+    let apiKey: String
+    let stopWords: [String]
+
+    init(
+        url: String? = nil,
+        modelName: String,
+        contextWindow: Int,
+        maxToken: Int,
+        temperature: Double = 0.2,
+        stopWords: [String] = [],
+        apiKey: String
+    ) {
+        self.url = url.flatMap(URL.init(string:)) ??
+            URL(string: "https://api.anthropic.com/v1/messages")!
+        self.modelName = modelName
+        self.maxToken = maxToken
+        self.temperature = temperature
+        self.apiKey = apiKey
+        self.stopWords = stopWords
+        self.contextWindow = contextWindow
+    }
+
+    public enum Models: String, CaseIterable {
+        case claude3Opus = "claude-3-opus-latest"
+        case claude35Sonnet = "claude-3-5-sonnet-latest"
+        case claude35Haiku = "claude-3-5-haiku-latest"
+
+        var maxToken: Int {
+            switch self {
+            case .claude3Opus: return 4096
+            case .claude35Sonnet: return 4096
+            case .claude35Haiku: return 4096
```

Comment:
I think all of them supports a 200_000 tokens context window.

Reply:
Right. Fixed! 😄